doorstop/cli/main.py

```diff
@@ -52,6 +52,8 @@ def main(args=None):  # pylint: disable=R0915
                         help="do not validate external file references")
     parser.add_argument('-C', '--no-child-check', action='store_true',
                         help="do not validate child (reverse) links")
+    parser.add_argument('-Z', '--strict-child-check', action='store_true',
+                        help="Strictly validate (reverse) links from every child document")
```

Comment:
To me, a help message like "require child (reverse) links from every document" is more clear.